packages/firestore/src/local/local_store.ts

```diff
@@ -666,6 +668,21 @@ export class LocalStore {
               viewChange.removedKeys,
               viewChange.targetId
             );
+
+            if (viewChange.fromCache) {
```

Comment:
I think this should be `!viewChange.fromCache`?

Reply:
Yes. Tests for this are coming soon :)